components/layout_2020/table/construct.rs

```diff
@@ -56,6 +56,19 @@ pub(crate) enum AnonymousTableContent<'dom, Node> {
     },
 }
 
+impl<'dom, Node> AnonymousTableContent<'dom, Node> {
+    fn is_whitespace_only(&self) -> bool {
+        match self {
+            Self::Element { .. } => false,
+            Self::Text(_, ref text) => text.chars().all(char_is_whitespace),
+        }
+    }
+
+    fn are_whitespace_only(contents: &[Self]) -> bool {
```

Comment:
Nit: `are` is a bit awkward here because it's not super clear from reading it that plurality matches first argument.

```suggestion
    fn contents_are_whitespace_only(contents: &[Self]) -> bool {
```